jobs/JREGIONAL_PROCESS_BUFR

```diff
@@ -70,13 +70,28 @@ mkdir_vrfy -p ${workdir}
 #
 #-----------------------------------------------------------------------
 #
+# create COMOUT directory
+#
+#-----------------------------------------------------------------------
+
+if [ "${RUN_ENVIR}" = "nco" ]; then 
+  if [ ${CYCLE_TYPE} == "spinup" ]; then
+    COMOUT="${COMOUT_BASEDIR}/$RUN.$PDY/${cyc}_spinup${SLASH_ENSMEM_SUBDIR}"
+  else
+    COMOUT="${COMOUT_BASEDIR}/$RUN.$PDY/${cyc}${SLASH_ENSMEM_SUBDIR}"
```

Comment:
If the COMOUT here contains the member subdir, the COMIN in process_lightning etc should have it as well.

Reply:
removed member ID based on Chunhua's comment